docs/source/extensions/strategy.rst

```diff
@@ -47,14 +63,74 @@ Training Strategies with Various Configs
     trainer = Trainer(accelerator="ipu", devices=8)
 
 
+The below table lists all relevant strategies available in Lightning with their corresponding short-hand name:
+
+
+.. list-table:: Strategy Classes and Nicknames
```

Comment:
I don't think so, because these are not unique strategies but rather variations of options. However, I do think we can mention them in the section for the strategy registry.